### PR TITLE
Search markup

### PR DESCRIPTION
**Pull request by skliarzfort**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <section class="search-section">
         <div class="container">
             <h1 class="search-section__title">Welcome to the Spryker <br>documentation center</h1>
-            <div class="search-section__panel">
+            <div class="search-section__panel dropdown-menu--dark">
```
Comment (tolerants) [inline _layouts/home.html:9]: According to BEM you can't use modifier class without base class. So please add `dropdown-menu` class.

File: js/customscripts.js [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+    $('#toc').toc({
+        minimumHeaders: 0,
+        listType: 'ul',
+        showEffect: 'none', // values: [show|slideDown|fadeIn|none]
+        headers: '.post-content h2, .post-content h3, .post-content h4'
+    });
+
+    /* this offset helps account for the space taken up by the floating toolbar. */
+    $('#toc').on('click', 'a', function() {
+        var target = $(this.getAttribute('href')),
```
Comment (tolerants) [inline js/customscripts.js:11]: As long as we drop Ie11 support suggest to avoid `var` usage.

File: scss/components/_anchor-nav.scss [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                &:hover {
+                    color: $darkest;
+                    border-left-color: $red;
+                }
+            }
+        }
+    }
+}
+
+.alt-nav {
```
Comment (tolerants) [inline scss/components/_anchor-nav.scss:47]: Please move into separate file.

File: scss/components/_icons.scss [hunk 4]
```diff
@@ -25,51 +25,54 @@
     -moz-osx-font-smoothing: grayscale;
 }
 
+.icon-github:before {
+  content: "\e910";
+}
 .icon-arrow-down:before {
-    content: "\e90c";
+  content: "\e90c";
```
Comment (tolerants) [inline scss/components/_icons.scss:32]: Why you changed indentions to 2 spaces? Please change back.

File: scss/main.scss [hunk 5]
```diff
@@ -7,10 +7,17 @@
...
+@import 'components/anchor-nav';
 @import 'pages/home';
+@import 'pages/inner';
 
 // temporary fix
 .navbar {
     display: none;
+}
+
+h1 {
```
Comment (tolerants) [inline scss/main.scss:21]: Please remove empty selector.

File: scss/main.scss [hunk 6]
```diff
@@ -7,10 +7,17 @@
...
 @import 'components/button';
 @import 'components/search-section';
+@import 'components/search-drop';
 @import 'components/cards';
+@import 'components/anchor-nav';
 @import 'pages/home';
+@import 'pages/inner';
 
 // temporary fix
 .navbar {
```
Comment (tolerants) [inline scss/main.scss:17]: Is still actual?

File: scss/pages/_inner.scss [hunk 7]
```diff
@@ -0,0 +1,65 @@
+.post-content {
```
Comment (tolerants) [inline scss/pages/_inner.scss:1]: Maybe `_content` will be a better file name? Or `_post`?

File: scss/pages/_inner.scss [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+        flex: 0 0 auto;
+        margin: 0 0 0 20px;
+
+        &:after {
+            content: none !important;
+        }
+    }
+}
+
+.confirm-block {
```
Comment (tolerants) [inline scss/pages/_inner.scss:37]: Please move to a separate file.

**Review by tolerants — CHANGES_REQUESTED** [review]

File: scss/pages/_inner.scss (hunk 7 above)
Comment (skliarzfort) [reply scss/pages/_inner.scss:1]: i'll change file name in the pr#74

**Review by tolerants — APPROVED** [review]

**Pull request merged**